### PR TITLE
Add PRESERVE_PATH setting to forword request_uri

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -5,7 +5,14 @@ cat > /etc/nginx/conf.d/default.conf <<EOF
...
     listen 80;
     server_name _;
-    return 301 "$REDIRECT_URL";
+    if ($PRESERVE_PATH != "True")
+      {
+        return 301 "$REDIRECT_URL";
+      }
+    if ($PRESERVE_PATH == "True")
+      {
+        return 301 "$REDIRECT_URL$request_uri";
```
Comment (bearnard) [inline entrypoint.sh]: this is all in a bash EOF context so if you want to keep `$` you need to escape it with a `\`

Maybe it's better to set `$REDIRECT_URL` before the EOF stuff then you don't need ifs in the nginx config either

Comment (KaitCrawford) [reply entrypoint.sh]: @bearnard any idea why the old code didn't need to be escaped?

Comment (KaitCrawford) [reply entrypoint.sh]: nevermind, I understand now

**Comment by bearnard** [comment]
👍

**Pull request merged**